std/machines/split/split_gl_vec.asm

```diff
@@ -0,0 +1,78 @@
+use std::array;
+use std::machines::large_field::memory::Memory;
+use super::split_gl::SplitGL;
+
+machine SplitGLVec8(mem: Memory, split_gl: SplitGL) with
```

Comment:
This is a question for someone with a better intuition on execution cost (@georgwiese): if instead of a fixed 8 element machine like this, would it be better if it took an N argument, and executed one split per row?

Reply:
I think it should be about the same